src/main/java/com/ekart/model/Size.java

```diff
@@ -28,11 +28,7 @@
     @Column(unique = true)
     private String size;
 
-    @OneToMany(mappedBy = "size", cascade = CascadeType.ALL)
-    @JsonIgnore
-    private Set<SubCategory> subCategories;
-
-    @OneToMany(mappedBy = "size", cascade = CascadeType.ALL)
+    @ManyToMany(mappedBy = "size", cascade = CascadeType.ALL)
```

Comment:
why manytomany?

Reply:
did not check the diff, will change it to onetomany only

Reply:
updated